.travis.yml

```diff
@@ -24,17 +24,15 @@ jobs:
       script: make build
 
     - stage: Test
-      name: GoLint
+      name: GoLint & Test
```

Comment:
I'd just call this job `Go`, and the js one `Javascript`. Since they're already part of the `Test` stage, it looks a bit redundant:

<img width="376" alt="Screen Shot 2019-06-20 at 12 37 05 PM" src="https://user-images.githubusercontent.com/2405957/59865887-32c9c580-9358-11e9-9519-c46e9a3aa5f6.png">